e2e/tests/init_test.go

```diff
@@ -912,6 +934,84 @@ func (s *suite) getServiceComputeUnitsPerRelay(serviceId string) uint64 {
 	return resp.Service.ComputeUnitsPerRelay
 }
 
+// getSharedParams returns the shared module parameters
+func (s *suite) getSharedParams() sharedtypes.Params {
+	args := []string{
+		"query",
+		"shared",
+		"params",
+		"--output=json",
+	}
+	res, err := s.pocketd.RunCommandOnHost("", args...)
+	require.NoError(s, err, "error querying shared params")
+
+	var sharedParamsRes sharedtypes.QueryParamsResponse
+
+	s.cdc.MustUnmarshalJSON([]byte(res.Stdout), &sharedParamsRes)
+	require.NoError(s, err)
+
+	return sharedParamsRes.Params
+}
+
+// getCurrentBlockHeight returns the current block height
+func (s *suite) getCurrentBlockHeight() int64 {
+	args := []string{
+		"query",
+		"block",
+		"--output=json",
+	}
+	res, err := s.pocketd.RunCommandOnHost("", args...)
+	require.NoError(s, err, "error querying for the latest block")
+
+	// Remove the first line of the response to avoid unmarshalling non JSON data
+	stdoutLines := strings.Split(res.Stdout, "\n")
```

Comment:
Why is this necessary given you're using --output=json?

Reply:
```
	// Remove the first line of the response to avoid unmarshalling non JSON data
	// since the query block command returns "Falling back to latest block height:"
	// as the first line when no height is provided.
```